server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -127,7 +127,15 @@ public enum MetadataVersion {
     // *** SHARE GROUPS BECOME PRODUCTION-READY IN THE FUTURE. ITS DEFINITION ALLOWS A SHARE   ***
     // *** GROUPS FEATURE TO BE DEFINED IN 4.1 BUT TURNED OFF BY DEFAULT, ABLE TO BE TURNED ON ***
     // *** DYNAMICALLY TO TRY OUT THE PREVIEW CAPABILITY.                                      ***
-    IBP_4_2_IV0(28, "4.2", "IV0", false);
+    IBP_4_2_IV0(28, "4.2", "IV0", false),
+
+    // Enables "streams" groups by default for new clusters (KIP-1071).
+    //
+    // *** THIS IS A PLACEHOLDER UNSTABLE VERSION WHICH IS USED TO DEFINE THE POINT AT WHICH     ***
+    // *** STREAMS GROUPS BECOME PRODUCTION-READY IN THE FUTURE. ITS DEFINITION ALLOWS A STREAMS ***
+    // *** GROUPS FEATURE TO BE DEFINED IN 4.1 BUT TURNED OFF BY DEFAULT, ABLE TO BE TURNED ON   ***
+    // *** DYNAMICALLY TO TRY OUT THE EARLY ACCESS CAPABILITY.                                   ***
+    IBP_4_2_IV1(29, "4.2", "IV1", false);
```

Comment:
It's a bit awkward to add an unstable MV for each unstable feature level. Could we introduce a single unstable MV that's shared between Qfk and streams MV?

Reply:
I did discuss this with @jolshan and her recommendation was to use two individual versions. I'll let her comment about it.

Reply:
I think it is easier to understand 1 new MV per feature. That way when your feature is ready you can mark it and you don't have to worry about other folks being ready etc.

Reply:
Thanks for the explanation, @jolshan. Sound good to me then.